### PR TITLE
ZAP CLA v1.0

### PR DESCRIPTION
**Pull request by psiinon**
Ready for review.
I plan to submit a PR for a blog post - the plan is to publish the blog post and link to this PR prior to merging it, so people have a chance to give feedback.

### CODE REVIEW

File: CLA.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## The Zed Attack Proxy (ZAP) Project - Individual Contributor License Agreement v1.0
+
+Thank you for your interest in the Zed Attck Proxy (ZAP) Project (the "Project"). In order to clarify the intellectual property license granted with Contributions from any person or entity, the Project must have a Contributor License Agreement ("CLA") on file that has been signed by each Contributor, indicating agreement to the license terms below. This license is for your protection as a Contributor as well as the protection of the Project and its users; it does not change your rights to use your own Contributions for any other purpose. Please read this document carefully before signing and keep a copy for your records.
```
Comment (ricekot) [inline CLA.md:3]:
```suggestion
Thank you for your interest in the Zed Attack Proxy (ZAP) Project (the "Project"). In order to clarify the intellectual property license granted with Contributions from any person or entity, the Project must have a Contributor License Agreement ("CLA") on file that has been signed by each Contributor, indicating agreement to the license terms below. This license is for your protection as a Contributor as well as the protection of the Project and its users; it does not change your rights to use your own Contributions for any other purpose. Please read this document carefully before signing and keep a copy for your records.
```

Comment (psiinon) [reply CLA.md]: Good spot! Fixed

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Review by ricekot — APPROVED** [review]
Thanks!

**Pull request merged**